django/db/migrations/questioner.py

```diff
@@ -185,35 +185,35 @@ def ask_not_null_alteration(self, field_name, model_name):
 
     def ask_rename(self, model_name, old_name, new_name, field_instance):
         """Was this field really renamed?"""
-        msg = "Did you rename %s.%s to %s.%s (a %s)? [y/N]"
+        msg = "Was %s.%s renamed to %s.%s (a %s)? [y/N]"
         return self._boolean_input(msg % (model_name, old_name, model_name, new_name,
                                           field_instance.__class__.__name__), False)
 
     def ask_rename_model(self, old_model_state, new_model_state):
         """Was this model really renamed?"""
-        msg = "Did you rename the %s.%s model to %s? [y/N]"
+        msg = "Was the model %s.%s renamed to %s? [y/N]"
         return self._boolean_input(msg % (old_model_state.app_label, old_model_state.name,
                                           new_model_state.name), False)
 
     def ask_merge(self, app_label):
         return self._boolean_input(
             "\nMerging will only work if the operations printed above do not conflict\n" +
             "with each other (working on different fields or models)\n" +
-            "Do you want to merge these migration branches? [y/N]",
+            "Should these migration branches be merged?",
             False,
         )
 
     def ask_auto_now_add_addition(self, field_name, model_name):
         """Adding an auto_now_add field to a model."""
         if not self.dry_run:
             choice = self._choice_input(
-                "You are trying to add the field '{}' with 'auto_now_add=True' "
-                "to {} without a default; the database needs something to "
+                "It is impossible to add the field '{}' with 'auto_now_add=True' "
+                "to {} without providing a default because the database needs something to "
```

Comment:
This sentence is too long, maybe:
```python
            choice = self._choice_input(
                f"It is impossible to add the field '{field_name}' with "
                f"'auto_now_add=True' to {model_name} without providing a "
                f"default. This is because the database needs something to "
                f"populate existing rows.\n",
                [
                ...
```